CuratedContent/2023-02-Inclusivescientificmeetings.md

```diff
@@ -0,0 +1,30 @@
+# Guidebook for Inclusive Scientific Meetings 
+
+<!-- deck text start --> 
+
+The Guide to Inclusive Scientific Meetings,  supports planners and managers in designing for successfuly inclusive scientific meetings.
+
+<!-- deck text start --> 
+
+#### Contributed by [Deborah Stevens](https://github.com/haikudeb)
+
+#### Publication date: Feb 2, 2023
+
+Resource information | Details 
+:--- | :--- 
+Resource name | 2023 Inclusive Scientific Meetings: Where to Begin
+Website | https://500womenscientists.org/inclusive-scientific-meetings
+Focus | Inclusivity
+
+The *Guide for Inclusive Scientific Meetings*, authored by the  *500 Women Scientists* organization, presents post-pandemic recommendations for getting the most out of planned scientific meetings/gatherings by maximizing inclusivity and equality. This updated 2023 guide (previously published in 2019) covers the range of meeting modalities -- in-person, hybrid, and virtual formats. The document begins with an overview of some excellent principles of inclusion before continuing into the main sections of 1) planning the meeting, 2) during the meeting, and 3) assessing the meeting. The Guide targets people in planning and leadership positions for scientific meetings and seeks to reframe the process of meeting planning with awareness and equitable participation.
+
+The pandemic years bought forth the importance of online meetings. This guide seeks to take the learning and best practices from the zoom/teams/screen interactions and move forward with a fresh range of meeting planning. This Guide aims to create “The art of the science meeting: how to do it really well—the fine points” from start to finish. 
```

Comment:
Rephrase the sentences " and move forward with a fresh range of meeting planning. This Guide aims to create “The art of the science meeting: how to do it really well—the fine points” from start to finish." for easier understanding

Reply:
> Rephrase the sentences for easier understanding " ....and move forward with a fresh range of meeting planning. This Guide aims to create The art of the science meeting: how to do it really well—the fine points from start to finish..."

I have rephrased and edited some segments of this article